src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyen_checkout/renderGenericComponent.js

```diff
@@ -107,6 +107,10 @@ module.exports.renderGenericComponent = async function renderGenericComponent()
     store.checkoutConfiguration.paymentMethodsResponse =
       data.AdyenPaymentMethods;
     setCheckoutConfiguration(data);
+    const amazonpayConfig = data.AdyenPaymentMethods.paymentMethods.find(
```

Comment:
If this configuration object in the `/paymentMethods` response applies to multiple payment methods: Can we make this generic so this will work with all payment methods that contain configuration data?

Reply:
N.B: Maybe a name like `paymentMethod` would be more descriptive than `elm`.